changelog.md

```diff
@@ -7,6 +7,34 @@ All notable end-user facing changes are documented in this file.
 *Here are all the changes in `master` branch, and will be moved to the appropriate release once they are included in a published nuget package.
 The idea is to track end-user facing changes as they occur.*
 
+### [2.0.0-beta2]
+
+- Known issues
+  - Code generation is too aggressive in generating serializers for most available types instead of just those that are directly used in grain methods. This causes excessive code being generated and compiled.
+
+- Breaking changes
+  - Remove `IGrainInvokeInterceptor` that got replaced with `IGrainCallFilter`
+  - Migrate more configuration settings to typed options
+  - Replace DeploymentId with ClusterId and collapse them where both are defined
+
+- Non-breaking improvements
+  - Better align silo hosting APIs with the future generic Microsoft.Extensions.Hosting.HostBuilder
+  - Multiple improvements to code generation
+  - Throw an exception when trying to build a silo with no application assemblies specified
+  - Multiple improvements to transactions
+  - Integrate Service Fabric clustering provider with `SiloHostBuilder`
+  - Split Service Fabric support assembly and NuGet package into two: for silo hosting and clustering
+  - Split `OrleansAzureUtils` assembly and NuGet package into more granular assemblies and packages
+  - Support for non-static serializers
+  - Add a timeout for synchronous socket read operations
+  - Support for multiple fallback serializers
+  - Enable TCP FastPath support
+  - Re-introduce run time code generation that can be enabled at silo host build time
+  - Support for Oracle in AdoNet (SQL) clustering provider
```

Comment:
I'm not sure I understand this correctly. In my recollection,  we support SQL membership and gatewaylist provider/clustering provider for long, we just renamed the feature to clustering provider. Is this not right?

Reply:
This line sounds like we supported new clustering provider

Reply:
This added support specifically for Oracle, in addition to already supported SQL Server, MySQL/MariaDB, and PostgreSQL.

Reply:
I see. Can we include the name for this new provider? easier for user to find it .

Reply:
It's not a separate provider, but rather support for Oracle in the already existing SQL/AdoNet provider. There are more details there than fit the release notes.